src/lesson02/Task05.java

```diff
@@ -0,0 +1,22 @@
+package lesson02;
+
+import java.util.Scanner;
+
+public class Task05 {
+    //programma perovodu inches to centimeters;
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        double inches;
+        System.out.print("Vvedit znachennya inches: " + "inches=");
```

Comment:
аналогічно тут, і такому виді ти створюєш два обєкта, це занадто
```suggestion
        double inches;
        System.out.print("Vvedit znachennya inches: inches=");
```